src/driver/communication_interface.cc

```diff
@@ -281,12 +288,18 @@ void CommunicationInterface::PublishRobotStatus() {
   if (std::unique_lock<std::mutex> lock {robot_data_mutex_};
       robot_data_.has_robot_data) {
     drake::lcmt_iiwa_status franka_status {
-        utils::ConvertToLcmStatus(robot_data_)};
+        utils::ConvertToLcmIiwaStatus(robot_data_)};
+
+    robot_msgs::robot_status_t robot_status {
+        utils::ConvertToRobotStatusLcmMsg(robot_data_)};
+
     // publish data over lcm
     franka::RobotMode current_mode {robot_data_.robot_state.robot_mode};
     robot_data_.has_robot_data = false;
     lock.unlock();
-    lcm_.publish(params_.lcm_status_channel, &franka_status);
+    lcm_.publish(params_.lcm_iiwa_status_channel, &franka_status);
+
+    lcm_.publish(params_.lcm_robot_status_channel, &robot_status);
```

Comment:
why both if one is a subset of the other?

Reply:
Because we can't remove IIWA status completely yet